docs/overview.rst

```diff
@@ -13,12 +13,13 @@ greater detail.
 Configuration
 ~~~~~~~~~~~~~
 
-After installing web3.py (via ``pip install web3``), you'll need to specify the
-provider and any middleware you want to use beyond the defaults.
+After installing web3.py (via ``pip install web3``), you'll need to specify
+async or sync web3, the provider, and any middleware you want to use
+beyond the defaults.
 
 
-Providers
----------
+Sync Providers
```

Comment:
I'd like this to be consistent with `AsyncProvider` below, but `SyncProviders` doesn't seem right. Maybe `Synchronous Providers` and `Asynchronous Providers`? Possibly a `Providers` header with a brief description of the differences?